modules/pool/lib/apr-data-sources/fantom/masterchef-farm-apr.service.ts

```diff
@@ -99,12 +99,16 @@ export class MasterchefFarmAprService implements PoolAprService {
             });
         }
 
-        (farm.rewarder?.rewardTokens || [])
-            .filter((rewardToken) => rewardToken.token !== networkConfig.beets.address)
-            .forEach((rewardToken) => {
+        if (farm.rewarder) {
+            for (const rewardToken of farm.rewarder?.rewardTokens) {
+                const farmRewarder = await prisma.prismaPoolStakingMasterChefFarmRewarder.findUniqueOrThrow({
+                    where: {
+                        id: `${farm.id.toLowerCase()}-${farm.rewarder?.id.toLowerCase()}-${rewardToken.token.toLowerCase()}`,
```

Comment:
that makes me cringe a bit. thats why it would have been better to use combined primary keys. but yea its what it is :)